package.json

```diff
@@ -19,7 +19,9 @@
     "stylus": "~0.54.5",
     "uglify-js": "^2.7.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "jsonlint": "^1.6.2"
```

Comment:
> who think that if it breaks anything

Doesn't seem to be particularly true for jsonlint, unless you are saying the exit code is not part of the API behavior: https://github.com/zaach/jsonlint/commit/b9a537a3c1c4c475b9be09502680ab5c95039edb